src/Compiler/FSComp.txt

```diff
@@ -236,7 +236,7 @@ forBadPrecision,"Bad precision in format specifier"
 forBadWidth,"Bad width in format specifier"
 forDoesNotSupportZeroFlag,"'%s' format does not support '0' flag"
 forPrecisionMissingAfterDot,"Precision missing after the '.'"
-forFormatDoesntSupportPrecision,"'%s' format does not support precision"
+forFormatDoesNotSupportPrecision,"'%s' format does not support precision"
```

Comment:
This rename will retrigger the whole translation pipeline which will take multiple weeks to translate in the best case scenario. Are we sure we want it?

Reply:
Actually I was going to create another PR (rebase https://github.com/dotnet/fsharp/pull/2558), but got distracted. I can drop this for if it will cause troubles.

Reply:
Yeah, it's better to not trigger it if no change to the message is introduced.